contrib/examples/apiserver/apiserver.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: service-catalog-apiserver
+  labels:
+    app: apiserver
+spec:
+  containers:
+  - name: apiserver
+    image: apiserver
+    args: 
+    - --etcd-servers
+    - http://localhost:2379
+    - -v
+    - "10"
+    ports:
+    - name: https
+      containerPort: 6443
+  - name: etcd
+    image: gcr.io/google_containers/etcd:3.0.14
```

Comment:
What did k8s core _"support"_ in December? Where will it be at 1.6?

 I have 3.0.9 on one machine, 3.0.7 on another, and the integration test script use `quay.io/coreos/etcd`. @duglin 

I think it's fine.